scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamBackend.scala

```diff
@@ -1,20 +1,33 @@
 package com.twitter.scalding.beam_backend
 
-import com.twitter.scalding.dagon.{FunctionK, Memoize, Rule}
 import com.twitter.chill.KryoInstantiator
 import com.twitter.chill.config.ScalaMapConfig
 import com.twitter.scalding.Config
-import com.twitter.scalding.beam_backend.BeamOp.{CoGroupedOp, MergedBeamOp}
+import com.twitter.scalding.beam_backend.BeamOp.CoGroupedOp
+import com.twitter.scalding.beam_backend.BeamOp.MergedBeamOp
+import com.twitter.scalding.dagon.FunctionK
+import com.twitter.scalding.dagon.Memoize
+import com.twitter.scalding.dagon.Rule
 import com.twitter.scalding.serialization.KryoHadoop
+import com.twitter.scalding.typed.CoGrouped.WithDescription
+import com.twitter.scalding.typed.OptimizationRules.AddExplicitForks
```

Comment:
nit: can we use a grouped import here rather than one-at-a-time java style:
```scala
import com.twitter.scalding.typed.OptimizationRules.{
  ComposeDescriptions, DeferMerge, Dia...
```

Reply:
Done. Thanks!